modules/gitops-creating-and-configuring-the-app-cr-to-reference-the-target-namespace-and-user-defined-appproject-instance.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * argocd_applications/managing-apps-in-non-control-plane-namespaces.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-creating-and-configuring-the-app-cr-to-reference-the-target-namespace-and-user-defined-appproject-instance_{context}"]
+= Creating and configuring the Application CR to reference the target namespace and user-defined AppProject instance
+
+As a cluster administrator, you can define a certain set of non-control plane namespaces in which users can create, update, and reconcile `Application` resources. After you configure the target namespaces in the `.spec.sourceNamespaces` field of the user-defined `AppProject` instance, you must explicitly create and configure the `Application` custom resource (CR) with the parameters for the `metadata.namespace` and `.spec.project` fields to reference the target namespace and user-defined `AppProject` instance.
+
+.Prerequisites
+* You are logged in to the {OCP} cluster as an administrator.
+* You have installed {gitops-title} 1.13.0 or a later version on your {OCP} cluster.
+
+.Procedure
+
+. Create and configure the `Application` CR with the parameters for the `metadata.namespace` and `.spec.project` fields to reference the target namespace and user-defined `AppProject` instance:
+.. From the *Project* list, select the target namespace.
+.. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators* -> *{gitops-title}* and go to the *Application* tab.
+.. Click *Create Application* and enter the following configuration in the YAML view:
++
+.Example user-defined `AppProject` instance
+[source,yaml]
+----
+kind: Application
+apiVersion: argoproj.io/v1alpha1
+metadata:
+  name: cluster-configs # <1>
+  namespace: dev # <2>
+spec:
+  project: project-one # <3>
+  # ...
+----
+<1> The name of the application.
+<2> The name of the target namespace for the Argo CD server to create and manage `Application` resources.
+<3> The name of the user-defined `AppProject` instance.
+.. Click *Create*.
++
+The *Applications* page displays the created application. 
+
+. Verify that the following role-based access control (RBAC) resources are created by the {gitops-shortname} Operator:
++
+[options="header"]
+|===
+|Name |Kind |Purpose
+|`<argocd_name>-<argocd_namespace>-argocd-application-controller` |`ClusterRole` and `ClusterRoleBinding` |For the Argo CD Application Controller to watch and list `Application` resources at cluster-level
+
+|`<argocd_name>-<argocd_namespace>-application` |`Role` and `RoleBinding` |For the Argo CD Application Controller to manage `Application` resources in target namespace
+
+|`<argocd_name>-<target_namespace>` |`Role` and `RoleBinding` |For the Argo CD server to manage `Application` resources in target namespace through the UI, API, or CLI
+|===
++
+[NOTE]
+====
+The Operator adds the `argocd.argoproj.io/application-managed-by-cluster-argocd` label to the target namespace.
+====
```

Comment:
This looks a little bit out of place to me. Can we move it to `Configuring the Argo CD CR...` section, maybe under ?
> 6. Verify that your user-defined cluster-scoped Argo CD instance is configured with a cluster role to manage cluster-scoped resources: